website/docs/guides/fusion-upgrade.md

```diff
@@ -0,0 +1,432 @@
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/start-upgrade.png" width="60%" title="Start the Fusion upgrade from the project homepage"/>
+
+You'll be redirected to the <Constant name="cloud_ide" /> with the upgrade assistant visible at the top.
+
+### Step 2: Check for deprecation warnings
+
+Even if you resolved deprecations in Part 1, run a final check to ensure nothing was missed:
+
+1. At the top of the <Constant name="cloud_ide" />, click **Check deprecation warnings**
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/check-deprecations.png" width="60%" title="Check for deprecation warnings in your project"/>
+
+2. Wait for the parse to complete (this may take a few moments depending on project size)
+3. Review the results:
+   - **No warnings found**: Skip to Step 4 to continue upgrading.
+   - **Warnings found**: Continue to Step 3 to resolve them.
+
+### Step 3: Resolve remaining deprecations
+
+If deprecation warnings are found, use the autofix tool to resolve them:
+
+1. In the deprecation warnings list, click **Autofix warnings**.
+2. Review the proposed changes in the dialog.
+3. Click **Continue** to apply the fixes automatically.
+4. Wait for the autofix tool to complete and run a follow-up parse.
+5. Review the modified files in the **Version control** panel.
+6. If all warnings are resolved, you'll see a success message.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/autofix-success.png" width="60%" title="Success message when deprecations are resolved"/>
+
+For detailed information about the autofix process, see [Fix deprecation warnings](/docs/cloud/dbt-cloud-ide/autofix-deprecations).
+
+:::info Manual fixes required?
+
+If the autofix tool can't resolve all deprecations automatically, you'll need to fix them manually. Review the warning messages for specific guidance, make the necessary changes in your code, then run **Check deprecation warnings** again.
+
+:::
+
+### Step 4: Enable Fusion
+
+Once all deprecations are resolved, upgrade your development environment:
+
+1. Click the **Enable Fusion** button at the top of the <Constant name="cloud_ide" />.
+2. Confirm the upgrade when prompted.
+3. Wait for the environment to update (this typically takes just a few seconds).
+
+Your development environment is now running on <Constant name="fusion" />!
+
+### Step 5: Restart the IDE
+
+After upgrading, all users need to restart their IDE to connect to the new <Constant name="fusion" />-powered environment:
+
+1. If you're currently in the <Constant name="cloud_ide" />, refresh your browser window.
+2. Notify your team members that they also need to restart their IDEs.
+
+### Step 6: Verify the upgrade
+
+Confirm your development environment is running <Constant name="fusion" />:
+
+1. Open or create a dbt model file in the <Constant name="cloud_ide" />
+2. Look for <Constant name="fusion" />-powered features:
```

Comment:
should we link to https://docs.getdbt.com/docs/fusion/supported-features#features-and-capabilities?